src/components/AttachmentCarousel/index.js

```diff
@@ -153,7 +153,7 @@ class AttachmentCarousel extends React.Component {
      */
     createInitialState() {
         const actions = [ReportActionsUtils.getParentReportAction(this.props.report), ...ReportActionsUtils.getSortedReportActions(_.values(this.props.reportActions))];
-        const attachments = [];
+        let attachments = [];
```

Comment:
Based on my comment below, there is no need to change this to `let` anymore I think.